lib/Logger.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
I think this way is best, this was just me being annoyed with the inability of using `export function` 💃